pyvista/core/utilities/features.py

```diff
@@ -36,14 +44,20 @@ def voxelize(mesh, density=None, check_surface=True):
     -------
     pyvista.UnstructuredGrid
         Voxelized unstructured grid of the original mesh.
+    pyvista.SructuredGrid
+        Voxelized volume (structured grid) that encloses input mesh.
 
     Notes
-    -----
+    -------
     Prior to version 0.39.0, this method improperly handled the order of
     structured coordinates.
-
+    
+    TODO: Add option to populate voxels with array elements based on
+    the approximate volume of the input mesh in each voxel.
+    e.g., input mesh volume comprises >50% of individual voxel volume
+    
     Examples
-    --------
+    ----------
```

Comment:
Did you mean to change this? The number of `-` should match the number of characters in te line above

Reply:
I did not mean to change this, apologies.